src/components/UseV2ExchangeModal/index.tsx

```diff
@@ -78,30 +82,34 @@ const UseV2ExchangeModal = ({ onDismiss = defaultOnDismiss }: UseV2ExchangeModal
             Go to V2 Exchange
           </Button>
         </StyledLink>
-        <Divider />
-        <StyledLabel htmlFor="acknowledgement">
-          <Flex alignItems="center" justifyContent="space-between">
-            <StyledCheckbox
-              id="acknowledgement"
-              checked={isAcknowledged}
-              onChange={() => setIsAcknowledged(!isAcknowledged)}
-              scale="sm"
-            />
-            <Text ml="16px" color={isAcknowledged ? 'text' : 'textDisabled'}>
-              I understand that V1 is no longer supported, and I may experience significant slippage, resulting in lost
-              capital.
-            </Text>
-          </Flex>
-        </StyledLabel>
-        <StyledButton
-          mt="24px"
-          width="100%"
-          variant="text"
-          disabled={!isAcknowledged || !hasTimerPassed}
-          onClick={onDismiss}
-        >
-          Continue to V1 Anyway {timerSecondsRemaining ? `(${timerSecondsRemaining})` : ''}
-        </StyledButton>
+        {isPageVisit && (
```

Comment:
Only render the bottom half of the modal with acknowledgement checkbox if `isPageVisit` is true